### PR TITLE
[127594] Move bugzilla calls to background script

### PR DESCRIPTION
**Pull request by mawaruDOLL**
Test comment for new pull request.

### CODE REVIEW

**Comment by mawaruDOLL** [comment]
Test comment for existing pull request.

**Comment by mawaruDOLL** [comment]
Test comment with time.

**Comment by mawaruDOLL** [comment]
I had to move and change a lot of code so there may be a few broken features but adding comments and time to a pull request page also updates quantum after the initial pull request is made.

**Comment by jtl3d** [comment]
Still testing. So far:
 - Labels aren't updating in Bugzilla.

File: src/background.js [hunk 1]
```diff
@@ -1,13 +1,179 @@
 /* Make the icon active when we're on our GitHub or Bugzilla pages */
```
Comment (jtl3d) [inline src/background.js]: This comment could be updated.

File: src/background.js [hunk 2]
```diff
@@ -1,13 +1,179 @@
...
+						response: response,
+						settings: request.bugzillaSettings,
+						bugId: request.bugId
+					});
+				});
+			break;
+
+		case "updateBugs":
+			bugzilla
+			.updateBugs(request.bug, request.params);
```
Comment (jtl3d) [inline src/background.js]: Weird whitespace.

**Comment by jtl3d** [comment]
Looks like the merge target was wrong when merging. I will address the issues I noted in a new branch.

**Pull request merged**